### DISCUSSION TITLE
How can I get data from the danger action?

### DISCUSSION THREAD

**Discussion by bdrelling** [fastlane/fastlane#18444] [Q&A]
I'd like to get the output of a [`danger`](https://docs.fastlane.tools/actions/danger/) run, check the # of warnings/errors/issues/etc., and perform some custom logic after the fact within my `Fastfile`.

As it stands, there is currently no `ENV` or lane context output from the command.

Anyone have any thoughts on doing this without modifying the action?

**Comment by rogerluan** [comment]
Hey 👋 

Not sure what are your use cases exactly, but _maybe_ your Dangerfile itself could write the results you're looking for into the disk, or into env vars. I don't think `danger` action will do what you want

**Reply by bdrelling** [reply]
Thanks for the reply!

My use case is, instead of having `danger` fail my build, I'd like to use the API of our CI to add an abort reason (eg. "Linting errors") for better reporting.

With that said, I'm currently trying to write env vars within the `Dangerfile`, but the moment the script stops executing those env vars seem to be gone. I think because `danger` is running in another process? 

I like the idea of writing to disk! That's smart.
